front50-test/src/main/java/com/netflix/spinnaker/front50/pipeline/SqlPipelineDAOTestConfiguration.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 Salesforce, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.front50.pipeline;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spectator.api.NoopRegistry;
+import com.netflix.spinnaker.config.Front50SqlProperties;
+import com.netflix.spinnaker.front50.config.StorageServiceConfigurationProperties;
+import com.netflix.spinnaker.front50.model.DefaultObjectKeyLoader;
+import com.netflix.spinnaker.front50.model.SqlStorageService;
+import com.netflix.spinnaker.front50.model.pipeline.DefaultPipelineDAO;
+import com.netflix.spinnaker.kork.sql.config.SqlRetryProperties;
+import com.netflix.spinnaker.kork.sql.test.SqlTestUtil;
+import io.github.resilience4j.circuitbreaker.CircuitBreakerRegistry;
+import java.time.Clock;
+import java.util.concurrent.Executors;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import rx.Scheduler;
+import rx.schedulers.Schedulers;
+
+@Configuration
```

Comment:
The groovy code didn't have this, nor the `@Bean` annotation on createPipelineDAO.  Tests pass for me locally without these.

Reply:
Yeah, true. Removed them now.